From de055b09de48b863c8c779fa6ac9cca59cbba07f Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <tgamblin@baylibre.com>
Date: Thu, 23 Jul 2026 08:54:31 -0400
Subject: [PATCH] Pillow: .ci: use 'uv pip'

We use uv in our workflows, so replace 'python3 -m pip' with 'uv pip' in
the .ci scripts run by the Pillow pipelines.

Upstream-Status: Inappropriate [different Python management tooling]

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 .ci/install.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/.ci/install.sh b/.ci/install.sh
index 516b310..a01d699 100755
--- a/.ci/install.sh
+++ b/.ci/install.sh
@@ -36,17 +36,17 @@ packages=(
 )
 sudo apt-get -qq install --no-install-recommends "${packages[@]}"
 
-python3 -m pip install --upgrade pip
-python3 -m pip install --upgrade ipython
+uv pip install --upgrade pip
+uv pip install --upgrade ipython
 
 # optional test dependencies, only install if there's a binary package.
-python3 -m pip install --only-binary=:all: numpy || true
-python3 -m pip install --only-binary=:all: pyarrow || true
+uv pip install --only-binary=:all: numpy || true
+uv pip install --only-binary=:all: pyarrow || true
 
 # PyQt6 doesn't support PyPy3
 if [[ $GHA_PYTHON_VERSION == 3.* ]]; then
     # pyqt6 doesn't yet support free-threading; only install if a wheel is available
-    python3 -m pip install --only-binary=:all: pyqt6 || true
+    uv pip install --only-binary=:all: pyqt6 || true
 fi
 
 # webp
-- 
2.55.0

